gallery/helper.cpp

```diff
@@ -56,7 +56,6 @@ qreal Helper::calculateScreenDpr()
 
 qreal Helper::calculateDpRatio()
 {
-#if MOBILE_OS
```

Comment:
why removed?

Reply:
To unify it with Input code. In gallery, `dp` was always set to 1, which made components look differently when used in Input vs gallery